parse-task-issue/util.js

```diff
@@ -0,0 +1,25 @@
+const jsyaml = require('js-yaml');
+const fs = require('fs');
+
+
+function getYMLFileContent(issue) {
+  let start = issue.body.indexOf("```")+3
+  let end = issue.body.lastIndexOf("```")
+  var newtask = jsyaml.load(issue.body.substring(start, end));
```

Comment:
should do some parsing here to filter out example entries and other junk

Reply:
Ok! this is handled in the last commit.